### ISSUE TITLE
Is it possible to applying Audio Monitor Filter to a track?

### ISSUE THREAD

**Issue by ysf** [exeldro/obs-audio-monitor#34]
Hey there, 

I'd like to know if it's possible to add a Filter a complete Track instead of having to add it to all the sources manually. That way also it would be possible to route sounds differently.

Thanks for your work!

**Comment by Mushiiies** [comment]
Yes, use audio monitor dock and then you can set a track to monitor

**Comment by ysf** [comment]
> Yes, use audio monitor dock and then you can set a track to monitor

Yes, I can set a track to monitor. But my question was if it is possible to **add a filter to a track**. That way I could add a lot of sounds to one track with the normal OBs Audio feature and re-route a complete track to some different output. This seems not possible in the Audio Monitor dock.

**Comment by Mushiiies** [comment]
In OBS, click view, docks - audio monitor.
In the audio monitor dock you can right click, then go to your tracks and
select a device to monitor the track

**Comment by ysf** [comment]
I'm really thankful for your patience. I'm sure this is what you mean:
![image](https://user-images.githubusercontent.com/34326/144488575-5fc14cf2-811d-4ca4-b1ed-18ed9793b4ee.png)
I could monitor a specific track on multiple outputs. I understood that.

I was stupid asking my question here, because what I meant was if it's possible to ADD A FILTER to a track. But this has nothing to do with your plugin and would be better assigned at general OBS.

Either way, thanks a lot for your work!

**Comment by Trouffman** [comment]
It is not possible to add filter to tracks specifically, ther are used for routing not mixing.
This is mostly a question for OBS directly rather than this plugin IMHO.
